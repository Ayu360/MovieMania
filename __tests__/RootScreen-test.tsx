import { render } from '@testing-library/react-native';

import RootIndex, { AppName } from '@/app/index';

describe('<RootIndex />', () => {
  test('Text renders correctly on RootIndex', () => {
    const { getByText } = render(<RootIndex />);

    getByText("MovieMania");
  });
  test('CustomText renders correctly', () => {
    const tree = render(<AppName>Some text</AppName>).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
