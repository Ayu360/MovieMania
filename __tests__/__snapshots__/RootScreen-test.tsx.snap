// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RootIndex /> CustomText renders correctly 1`] = `
<Text
  style={
    {
      "color": "#FF8E01",
    }
  }
>
  Some text
</Text>
`;
